docs/content/documentation/systems/Materials/tensor_mechanics/CappedMohrCoulombCosseratStressUpdate.md

```diff
@@ -0,0 +1,2 @@
+# CappedMohrCoulombCosseratStressUpdate
```

Comment:
What I would rather you add this file to the list of hidden class (see https://github.com/idaholab/moose/blob/devel/docs/hidden.yml#L908) and just delete this file. So when we start documenting you just work through the list to make sure everything is done.